source/includes/_object-reference.md

```diff
@@ -13,7 +13,7 @@ The Crunch API consists of just a few primitive objects, arranged differently fo
 ### Shoji entity
 
 A Shoji entity is identified by the `element` key having value `shoji:entity`.
-Its principal attribute is the `body` key wich is an object containing the 
+Its principal attribute is the `body` key wich is an object containing the
```

Comment:
Since your linter is already touching this line, can you please fix the grammar and spelling?